.github/workflows/update-gpuci.yml

```diff
@@ -0,0 +1,51 @@
+name: Check for gpuCI updates
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+  workflow_dispatch:
+
+jobs:
+  update-gpuci:
+    runs-on: ubuntu-latest
+    if: github.repository == 'dask/dask'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get latest cuDF nightly version
+        id: latest_version
+        uses: jacobtomlinson/gha-anaconda-package-version@0.1.3
+        with:
+          org: "rapidsai-nightly"
+          package: "cudf"
+          version_system: "CalVer"
+
+      - name: Strip git tags from versions
+        run: echo "RAPIDS_VER=${FULL_RAPIDS_VER::-10}" >> $GITHUB_ENV
+        env:
+          FULL_RAPIDS_VER: ${{ steps.latest_version.outputs.version }}
+
+      - name: Find and Replace Release
+        uses: jacobtomlinson/gha-find-replace@0.1.4
+        with:
+          include: 'continuous_integration\/gpuci\/axis\.yaml'
+          find: "RAPIDS_VER:\n- .*"
+          replace: |-
+            RAPIDS_VER:
+            - "${{ env.RAPIDS_VER }}"
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: "Update gpuCI `RAPIDS_VER` to `${{ env.RAPIDS_VER }}`"
+          title: "Update gpuCI `RAPIDS_VER` to `${{ env.RAPIDS_VER }}`"
+          reviewers: "charlesbluca"
+          # labels: ""
```

Comment:
Any suggestions on labels / reviewers? Would probably make sense to add some maintainers that are members of RAPIDS

Reply:
We can probably leave off labels for now. For reviewers, I'll suggest `dask/gpu` instead of specific individuals. This will ensure folks from RAPIDS are pinged.

Reply:
That makes sense, though it looks like we might need to use a non-default GitHub token when opening the PR in order to request review from a team:

https://github.com/peter-evans/create-pull-request/issues/155#issuecomment-611904487